Integrations/src/main/java/io/deephaven/integrations/learn/Output.java

```diff
@@ -0,0 +1,41 @@
+package io.deephaven.integrations.learn;
+
+import org.jpy.PyObject;
+
+/**
+ * This class provides an interface for converting Python deep learning output to Deephaven table columns.
+ * Output objects are intended to be used as the input argument of an eval() function call.
+ */
+
+public class Output {
+
+    final String[] colNames;
+    final PyObject func;
+    final String type;
+
+    /**
+     * Constructor for Output object.
+     *
+     * @param colNames The array of column names from a Deephaven table to be used in modelling.
+     * @param func     The function that determines how data from a Python object is scattered back to a Deephaven table.
+     * @param type     The datatype that the user wishes to return. Note this will probably only be useful if it's a
+     *                 primitive data type.
+     */
+    public Output(String[] colNames, PyObject func, String type) {
+        this.colNames = colNames;
```

Comment:
Some questions about good input validation design:

Suppose the user passes output argument to eval that looks like 
`output = [learn.Output(["Out1"], scatter, "int"), learn.Output(["Out1"], scatter, "int")]`.
Well, it seems that this should complain, and tell the user they cannot have two output columns of the same name. It will complain when it tries to actually add two columns of the same name to the table, but I suspect we want something to catch it before it gets to that point. However, validating each Output independently is not sufficient to handle this case, because each individual Output is perfectly valid on its own, but as a set they are not. So,
1. Do we let this problem proceed until the table can't add another column of the same name, and let it complain as it will? Probably not. We'll waste time gathering and going through model_func, only to throw an obvious error at the end.
2. Do we try to do all of the error handling in Java? I think this works for validating individual Outputs, but cannot be used to validate the set of Outputs, because that set gets deconstructed before it's ever passed to Java at all. Java is never made aware of a "set of Outputs", only Python knows about that.
3. So, do we have a layer of validation in Java that handles each Output within its class, and then a layer of validation in Python that handles the set of Outputs? This makes sense to me, but maybe there's a good reason to not do this.
4. If so, we'll have some things throwing Java errors and some throwing Python errors. I know as a user last summer, I was much happier to see a Python error, because I was working in Python. Just a thought.

Can apply this to Input as well; there may be possible problems in the "set of Inputs" that cannot be caught by any individual element. So again, a Java layer to handle each individual object,  and a Python layer to handle the set. I've already got a method that looks at the set of Inputs and does stuff, and currently there is some error handling that may or may not stay there.

I guess this boils down to asking where the validation logic does and does not belong, and if it's appropriate to have multiple layers of validation for handling higher and lower level data structures.